Rendera/src/main.cpp

```diff
@@ -42,9 +43,17 @@ color lighting(Material mat, ray cam_ray, ray normal, vec3 point, vec3 light_src
 }
 
 int main(){
-    
+    //initialise a random number generator
+    std::minstd_rand random;
+
     vec3 center = {5,0,0};
-    color blue = {1,0.2, 1};
+<<<<<<< HEAD
```

Comment:
It appears that there was a merge conflict here that we failed to reconcile. We should fix that.